<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="16004" started-at="2019-06-21T10:46:25Z" finished-at="2019-06-21T10:46:41Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="16004" started-at="2019-06-21T10:46:25Z" finished-at="2019-06-21T10:46:41Z">
      <class name="com.training.regression.tests.MultipleUsers_Test">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:com.training.regression.tests.MultipleUsers_Test@3e6fa38a]" name="setUpBeforeClass" is-config="true" duration-ms="39" started-at="2019-06-21T10:46:25Z" finished-at="2019-06-21T10:46:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.training.regression.tests.MultipleUsers_Test@3e6fa38a]" name="setUp" is-config="true" duration-ms="9459" started-at="2019-06-21T10:46:26Z" finished-at="2019-06-21T10:46:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginDBTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.training.regression.tests.MultipleUsers_Test@3e6fa38a]" name="loginDBTest" duration-ms="5661" started-at="2019-06-21T10:46:35Z" data-provider="excel-inputs" finished-at="2019-06-21T10:46:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[manzoor6]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[mehadi]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[manzoo6@gmail.com]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[manzvenu]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[manzoor6]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[manzoor6]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[9876543210]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [] but found [Registration]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [] but found [Registration]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.training.regression.tests.MultipleUsers_Test.loginDBTest(MultipleUsers_Test.java:88)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDBTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.training.regression.tests.MultipleUsers_Test@3e6fa38a]" name="tearDown" is-config="true" duration-ms="4" started-at="2019-06-21T10:46:41Z" finished-at="2019-06-21T10:46:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.training.regression.tests.MultipleUsers_Test -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
